streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -534,11 +543,13 @@ public void shouldFetchAndIterateOverKeyRange() {
         );
     }
 
+    @SuppressWarnings("deprecation")
     @Test(expected = NullPointerException.class)
     public void shouldThrowNullPointerExceptionOnPutNullKey() {
         cachingStore.put(null, bytesValue("anyValue"));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Same with this, we are actually testing the deprecated method here, so we can justify the suppression.

Reply:
`
  @SuppressWarnings("deprecation")// continuing to support WindowStore#put(K, V) as internal api, this call uses the 3 params
    @Override
    public synchronized void put(final Bytes key,
                                 final byte[] value) {
        put(key, value, context.timestamp());
    }
`

Deprecation has been removed from CachingWindowStore.class